xtask/src/xtask.rs

```diff
@@ -186,14 +186,26 @@ fn test_tmt(sh: &Shell) -> Result<()> {
                 Ok(acc)
             })?;
     all_plan_files.sort_by_key(|v| v.0);
-    println!("Discovered plans: {all_plan_files:?}");
+    Ok(all_plan_files)
+}
+
+#[context("test-integration")]
+fn test_tmt(sh: &Shell) -> Result<()> {
+    let tests = match std::env::var("TMT_TEST") {
+        Ok(test) => {
+            vec![(0, test)]
+        }
+        Err(_) => all_plan_files(sh)?,
+    };
```

Comment:
I think this can be `let tests = std::env::var("TMT_TEST").unwrap_or_else(|| vec![(0, test]));`